README.md

```diff
@@ -39,4 +41,4 @@ If you are interested in working on the implementation for an 'active' RFC, but
 
 ## Reviewing RFCs
 
-This is where we differ from other teams using this process — we don't have regular core team meetings, and tend to do all our thinking informally, in the open, when time allows.
\ No newline at end of file
+This is where we differ from other teams using this process — we don't have regular core team meetings, and tend to do all our thinking informally, in the open, when time allows.
```

Comment:
This should probably be updated, but I'm not sure to what.

Reply:
good catch! I took a stab at updating this